### ISSUE TITLE
Update WAFS to build/run with hpc-stack library modules

### ISSUE THREAD

**Issue by KateFriedman-NOAA** [NOAA-EMC/EMC_gfs_wafs#2]
Update EMC_gfs_wafs to build and run with the hpc-stack (v1.1.0) on supported platforms. 

Companion issue on global-workflow side is https://github.com/NOAA-EMC/global-workflow/issues/164

Questions on using the stack can go to the developers who maintain it here:

https://github.com/NOAA-EMC/hpc-stack

**Comment by YaliMao-NOAA** [comment]
The master branch is updated to hpc-stack version.

**Comment by YaliMao-NOAA** [comment]
Now on Dell, there is an issue when using wgrib2 to convert grib2 data. No issue on Hera.

Since wgrib2 is loaded in a job card, the issue is not caused by the WAFS package itself.

The issue is wgrib2 of module wgrib2/2.0.8 under hpc-stack doesn't work when converting a grid file, as reported https://github.com/NOAA-EMC/hpc-stack/issues/224

For the test runs on Dell, I used wgrib2 of grib_util in the job card of wafs_grib2 and wgrib2 of hpc-stack in all other job cards. The outputs looked good.
For the test runs on Hera, I used wgrib2 of hpc-stack. All jobs went through and the outputs looked good too.
